### ISSUE TITLE
minor - align overviews (in foundational skills chapters) with introductions (in the walkthroughs)?

### ISSUE THREAD

**Issue by jrosen48** [data-edu/data-science-in-education#321]
Both foundational skills chapters have nice overviews, i.e., foundational skills 1:

```
We will be covering the following topics in this chapter: 

- Installing R and RStudio
- RStudio environment and pane layout
- Basics of customizing your RStudio environment
- Steps for working through new and unfamiliar content
- Accessing the data sets used in this book
```

Foundational skills 2:

```
This chapter is designed to give you the skills and knowledge necessary to *get
started* in any of the walk through chapters. Our goal is to get you working
with R using the RStudio *I*ntegrated *D*evelopment *E*nvironment (IDE) through
a series of applied examples. If you have not yet installed R and/or RStudio,
please go through the steps outlined in Chapter 05 before beginning this
chapter.
````

At the same time, the walkthroughs have introductions (as a part of the walkthrough templates), i.e. the gradebooks walkthrough:

```
There are a variety of data sources to explore in the education field. Student assessment scores can be examined for progress towards goals. The text from a teacher’s written classroom observation notes about a particular learner’s in-class behavior or emotional status can be analyzed for trends. We can tap into the exportable data available from common learning software or platforms popular in the K-12 education space.
```

Should we try to align these a bit more - in name (overview/introduction) and perhaps what's included in them?

**Comment by ivelasq** [comment]
I like this idea! Agree with aligning intro/overview, I can work on that.

**Comment by jrosen48** [comment]
I think this might be addressed - is that right I can do a quick review to make sure.

**Comment by jrosen48** [comment]
@ivelasq I can finalize this today, if this sounds good

**Comment by jrosen48** [comment]
can do at the same time as adding functions introduced/topics covered as a part of #424

**Comment by jrosen48** [comment]
addressed in #484
